server/svix-server/Cargo.toml

```diff
@@ -72,7 +72,7 @@ urlencoding = "2.1.2"
 form_urlencoded = "1.1.0"
 lapin = "2.1.1"
 sentry = { version = "0.32.2", features = ["tracing"] }
-omniqueue = { git = "https://github.com/svix/omniqueue-rs.git", rev = "044fc688670759859f193bb646fac043bbfd08ba", default-features = false, features = ["in_memory", "rabbitmq", "redis_cluster"] }
+omniqueue = { version = "0.2.0", default-features = false, features = ["in_memory", "rabbitmq-with-message-ids", "redis_cluster"] }
```

Comment:
@svix-jplatte, why is this feature not just called `rabbitmq`?

Reply:
We broke compatibility in a few ways in this version anyway, no? So why not break (read: fix) this too?

Reply:
We broke runtime compatibility with omniqueue 0.1 for redis to preserve compatibility between svix'es redis backend and the omniqueue redis backend (the omniqueue 0.1 behavior didn't really make sense). This feature is also for compatibility with the pre-omniqueue queue in svix, but it's not useful for other omniqueue users (unlike the redis change).